osu.Game/Rulesets/Mods/Mod.cs

```diff
@@ -136,19 +133,35 @@ public virtual Mod CreateCopy()
             // Copy bindable values across
             foreach (var (_, prop) in this.GetSettingsSourceProperties())
             {
-                var origBindable = prop.GetValue(this);
-                var copyBindable = prop.GetValue(copy);
-
-                // The bindables themselves are readonly, so the value must be transferred through the Bindable<T>.Value property.
-                var valueProperty = origBindable.GetType().GetProperty(nameof(Bindable<object>.Value), BindingFlags.Public | BindingFlags.Instance);
-                Debug.Assert(valueProperty != null);
+                var origBindable = (IBindable)prop.GetValue(this);
+                var copyBindable = (IBindable)prop.GetValue(copy);
 
-                valueProperty.SetValue(copyBindable, valueProperty.GetValue(origBindable));
+                // we only care about changes that have been made away from defaults.
+                if (!origBindable.IsDefault)
+                    copy.CopyAdjustedSetting(copyBindable, origBindable);
             }
 
             return copy;
         }
 
+        /// <summary>
+        /// When creating copies or clones of a Mod, this method will be called to copy explicitly adjusted user settings.
+        /// The base implementation will transfer the value via <see cref="Bindable{T}.Parse"/> and should be called unless replaced with custom logic.
+        /// </summary>
+        /// <param name="bindable">The target bindable to apply the adjustment.</param>
+        /// <param name="value">The adjustment to apply.</param>
+        internal virtual void CopyAdjustedSetting(IBindable bindable, object value)
+        {
+            if (value is IBindable incoming)
+            {
+                //copy including transfer of default values.
+                bindable.BindTo(incoming);
+                bindable.UnbindFrom(incoming);
```

Comment:
missing the fact that the recent framework PR that got merged did not transfer default/disabled was a huge oversight on my part, but i think i'm just going to look away for now to not complicate matters even more than i already have. hopefully most/all of this will get torn out at some point anyway.

Reply:
i'm not 100% sure a call to `Parse` *should* transfer more than value though. that's up for further discussion.

Reply:
yeah, I've intentionally made `Parse` not copy default and disabled because of the above reason, it shouldn't be doing so.